modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/mobile-menu-buttom/index.css

```
@@ -0,0 +1,11 @@
+.show-menu {
+	height: 200px;
+	opacity: 1;
+	transition: height 0.6s , opacity 0.8s;
+}
+
+.hidden-menu {
+	height: 0px;
```

If any CSS property is 0 it is not necessary to put the type of measurement unit (px, rem, vh...)

```suggestion
	height: 0;
```